resources/lrsql/config/prod/default/webserver.edn

```diff
@@ -12,4 +12,5 @@
  :http-port         #long #or [#env LRSQL_HTTP_PORT 8080]
  :ssl-port          #long #or [#env LRSQL_SSL_PORT 8443]
  :url-prefix        #or [#env LRSQL_URL_PREFIX "/xapi"]
- :enable-admin-ui   #boolean #or [#env LRSQL_ENABLE_ADMIN_UI true]}
+ :enable-admin-ui   #boolean #or [#env LRSQL_ENABLE_ADMIN_UI true]
+ :enable-stmt-html  #boolean #or [#env LRSQL_ENABLE_STMT_HTML false]}
```

Comment:
The docs include the statement browser, which uses this. You want to default it false?

Reply:
Oh yeah forgot it was actually default `true`.

Reply:
https://github.com/yetanalytics/lrsql/pull/185/commits/e7bc07ea9db5bd34119b51a9c65dbdf992b42fa4